types/experiments.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * The experiment definition accessible to Firefox via Remote Settings.
+ * It is compatible with ExperimentManager.
+ */
+export interface ExperimentRecipe {
+  /** A unique identifier for the Recipe */
+  id: string;
+  /** JEXL expression defined in an Audience */
+  filter_expression: string;
+  /** Experiment definition */
+  arguments: Experiment;
+}
+
+export interface Experiment {
+  /** Unique identifier for the experiment */
+  slug: string;
+  /** Publically-accesible name of the experiment */
+  public_name: string;
+  /** Short public description of the experiment */
+  public_description: string;
+  /** Experimenter URL */
+  experiment_url: string;
+  /** Is the experiment currently live in production? i.e., published to remote settings? */
+  is_published: boolean;
+  /** Are we continuing to enroll new users into the experiment? */
+  isEnrollmentPaused: boolean;
```

Comment:
Q: Why did this one change to a non-standard camelCase vs lowercase_underscore?

Reply:
After a long discussion I've changed everything to camelCase – in this case, this is for comaptibility with existing firefox code